package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@bitmovin/player-integration-comscore",
+  "name": "@bitmovin/player-integration-comscore2",
```

Comment:
What was the intention to rename this package but include the old version of this package as a dependency again?

That does not seem right at all.